trailblazer/store/crud/update.py

```diff
@@ -32,6 +32,19 @@ def update_user_is_archived(self, user: User, archive: bool = True) -> None:
         user.is_archived = archive
         self.commit()
 
+    def update_ongoing_analyses(self, analysis_host: Optional[str] = None) -> None:
+        """Iterate over all analysis with ongoing status and query SLURM for current progress."""
+        ongoing_analyses: Optional[List[Analysis]] = self.get_analyses_with_statuses(
+            statuses=list(TrailblazerStatus.ongoing_statuses())
+        )
+        for analysis in ongoing_analyses:
+            try:
+                self.update_run_status(analysis_id=analysis.id, analysis_host=analysis_host)
+            except Exception as error:
```

Comment:
```suggestion
            except Exception as error:
```
Can we make the scope of this exception smaller?

Reply:
Would be good - but out of scope